instrumentation/apache-pulsar/apache-pulsar-2.8/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pulsar/v28/StringTuple2.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.pulsar.v28;
+
+public class StringTuple2 {
+  public final String f1;
+  public final String f2;
```

Comment:
since only used in one place, could make it clearer
```suggestion
public class ProducerData {
  public final String url;
  public final String topic;
```